models/config.go

```diff
@@ -99,6 +101,23 @@ func (c ConfigChange) String() string {
 	return fmt.Sprintf("[%s/%s] %s", c.ExternalType, c.ExternalID, c.ChangeType)
 }
 
+// BeforeCreate is a user defined hook for Gorm.
+// It will be called when creating a record.
+func (c *ConfigChange) BeforeCreate(tx *gorm.DB) (err error) {
+	if c.ID == "" {
+		c.ID = uuid.New().String()
+	}
+
+	if c.CreatedAt == nil {
+		now := time.Now()
```

Comment:
I don't think we can use server time  - it needs to use the db time = it should do it already with the db default value ?

Reply:
@moshloop that's correct - should be populated in DB. Not sure why it was like that before. I just copy-pasted this `BeforeCreate` hook from config db models as I'm getting rid of the db models from there

https://github.com/flanksource/config-db/blob/main/db/models/config_change.go#L63-L74

Reply:
Made the change